### PR TITLE
Mod rewrite

### PR DESCRIPTION
**Pull request by LoCrealloc**
**Description**

Commands for: 
- editing warns, kicks, mutes, bans

More user log entries:
- mo deletions
- verification

Functionality for:
Using other time periods than days for mutes/bans. Now you can also use:
- years
- months
- weeks
- hours
- minutes

**Additional Notes**
You need to change a few database tables. Affected tables:
- warn
- mute
- kick
- ban

### CODE REVIEW

File: morpheushelper/cogs/mod.py [hunk 1]
```diff
@@ -14,22 +14,61 @@
...
+
+    @staticmethod
+    def convert(values: List[int]) -> int:
+        mins = values[5]
+        mins += ToMinutes.from_years(values[0])
+        mins += ToMinutes.from_months(values[1])
+        mins += ToMinutes.from_weeks(values[2])
+        mins += ToMinutes.from_days(values[3])
+        mins += ToMinutes.from_hours(values[4])
+        return mins
```
Comment (ghost) [inline morpheushelper/cogs/mod.py:71]: I think this is better of in a util package

File: morpheushelper/cogs/mod.py [hunk 2]
```diff
@@ -98,6 +137,15 @@ async def send_to_changelog_mod(
     await send_to_changelog(guild, embed)
 
 
+def check_reason_length(reason):
+    if len(reason) > 900:
+        raise CommandError(translations.reason_too_long)
+
+
+def minutes_to_days(minutes) -> int:
+    return (minutes / 60) / 24
```
Comment (ghost) [inline morpheushelper/cogs/mod.py:146]: Maybe this should also be a Time util

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by Paul-Brauckmann** [comment]
Due to a the restructuring of the Bot this pull reuest is not up to date. Feel free to close this and reopen it in the [cogs-repo](https://github.com/PyDrocsid/cogs).

**Review by ghost — CHANGES_REQUESTED** [review]
Due to a the restructuring of the Bot this pull reuest is not up to date. Feel free to close this and reopen it in the [cogs-repo](https://github.com/PyDrocsid/cogs).

**Pull request closed without merging**